### PR TITLE
Correct License shortcode. This stops NPM warning about valid SPDX license expression

### PR DESCRIPTION
**Pull request by omefire**
This PR stops npm from complaining about SDPX license expression during livereload dependencies installation:
"npm WARN package.json cordova-plugin-livereload@1.0.0 license should be a valid SPDX license expression"

### CODE REVIEW

**Comment by omefire** [comment]
@nikhilkh to review

File: package.json [hunk 1]
```diff
@@ -5,21 +5,19 @@
   "description": "Cordova Plugin to facilitate livereload workflow",
   "main": "lib/index.js",
   "author": "Omar Mefire <omefire@gmail.com>",
-  "license": "Apache 2.0",
+  "license": "Apache-2.0",
```
Comment (nikhilkh) [inline package.json]: Btw, our license should be MIT.

Comment (omefire) [inline package.json]: Updated.

**Comment by omefire** [comment]
LICENSE file udpated: https://github.com/omefire/cordova-plugin-livereload/commit/0d240d5143450db98ee4bef4a73cd47a057b9415

**Pull request merged**